frontend/src/components/window/Tab.js

```diff
@@ -20,18 +20,14 @@ class Tab extends PureComponent {
     } = this.props;
 
     if (docId && queryOnActivate) {
-      const query = orderBy
-        ? (orderBy[0].ascending ? '+' : '-') + orderBy[0].fieldName
-        : '';
-
       if (singleRowView) {
-        fetchTab({ tabId, windowId, docId, query }).then((res) => {
+        fetchTab({ tabId, windowId, docId, query: orderBy }).then((res) => {
```

Comment:
pls consider renaming `fetchTab` query param to `orderBy`. Saw that there are just 3 places where it's used (including this one).